Add a link to the code sample in the nodejs example (#251)
neondatabase/website: content/docs/integrations/node.md

  - /docs/quickstart/node
---

> This guide describes how to create a Neon project and connect to it from a Node.js application. You can find the below code sample on [GitHub](https://github.com/neondatabase/examples/tree/main/with-nodejs).

Reviewer: ```suggestion
This guide describes how to create a Neon project and connect to it from a Node.js application. The code samples in this guide are also available on [GitHub](https://github.com/neondatabase/examples/tree/main/with-nodejs).
```
